recipes/longshot/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -e
+
+# this build script is taken from the rust-bio-tools recipe
+# https://github.com/bioconda/bioconda-recipes/blob/master/recipes/rust-bio-tools/build.sh
+
+# taken from yacrd recipe, see: https://github.com/bioconda/bioconda-recipes/blob/2b02c3db6400499d910bc5f297d23cb20c9db4f8/recipes/yacrd/build.sh
+if [ "$(uname)" == "Darwin" ]; then
+
+    # apparently the HOME variable isn't set correctly, and circle ci output indicates the following as the home directory
+    export HOME="/Users/distiller"
+
+    # according to https://github.com/rust-lang/cargo/issues/2422#issuecomment-198458960 removing circle ci default configuration solves cargo trouble downloading crates
```

Comment:
I'll show my ignorance of rust here (I really should start a project using it), but what dependencies or other crates does this recipe require? Ideally we'd make those conda packages too (assuming that makes any sense at all for rust).

Reply:
The recipe requires a whole host of crates, but they are managed automatically by cargo. I don't think it's feasible to have them be managed by conda instead. I based this recipe off Johannes' rust-bio-tools recipe (very little is changed, it basically specifies some non-rust dependencies and calls ```cargo install```).

Reply:
Rust statically compiles pure-rust dependencies stuff into the binary. This is all fine. Those deps usually very small.